tools/decode-stacktrace.py

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/python2
```

Comment:
even more proper way is to use 
#!/usr/bin/env python
invocation of python. It is more platform independent then hardcoded path